src/createServer.js

```diff
@@ -1,3 +1,39 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { checkUrlParams } = require('./checkUrlParams');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const urlParams = new URL(req.url, `http://${req.headers.host}`);
+    const textToConvert = urlParams.pathname.slice(1);
+    const toCase = urlParams.searchParams.get('toCase');
+
+    const errors = checkUrlParams(textToConvert, toCase);
+
+    if (errors.length > 0) {
+      res.statusCode = 400;
+      res.statusMessage = 'Bad request';
+      res.end(JSON.stringify({ errors }));
+    } else {
+      const result = convertToCase(textToConvert, toCase);
```

Comment:
Is it necessary to use `else` here? 
You can add the return to the if statement and use code under as a default

```suggestion
    if (errors.length > 0) {
      res.statusCode = 400;
      res.statusMessage = 'Bad request';
      res.end(JSON.stringify({ errors }));
      return;
    } 

    const result = convertToCase(textToConvert, toCase);
```